pt-br/overviews/collections/introduction.md

```diff
@@ -0,0 +1,41 @@
+
+**Fáceis de usar:** Um pequeno vocabulário de 20 a 50 métodos é o suficiente para resolver a maioria dos problemas envolvendo coleções em poucas operações. Não é preciso aplicar estruturas de iteração ou recursão complicadas. Coleções persistentes e operações sem efeitos colaterais fazem com que você não precise se preocupar com corromper acidentalmente coleções existentes com dados novos. Interferência entre _iterators_ e atualizações de coleções são eliminadas.
+
+**Concisas:** Você pode fazer com apenas uma palavra o que levaria um ou vários loops. Você pode expressar operações funcionais com sintaxe leve e combinar operações sem esforço, de forma que o resultado se parece com uma álgebra adaptada.
+
+**Seguras:** Esta tem que ser experimentada para ser absorvida. A natureza estaticamente tipada e funcional das coleções Scala faz com que a vasta maioria dos erros que você pode vir a cometer seja capturada durante a compilação. A motivo disso é que (1) as operações das coleções em si são amplamente usadas e portanto bem testadas, (2) os usos das operações de coleções tem entradas e saídas explícitas na forma de parâmetros e retornos de funções, e (3) estas entradas e saídas explícitas são sujetas à verificação estática de tipo. Em síntese, a grande maioria dos usos incorretos se manifestam como erros de tipo. Não é totalmente incomum ver programas de centenas de linhas executar na primeira tentativa.
+
+**Rápidas:** Operações de coleções são ajustadas e otimizadas em suas bibliotecas. E o resultado é que o uso de coleções é normalmente bastante eficiente. Talvez você consiga resultados um pouco melhores com estruturas de dados e operações cuidadosamente ajustadas, mas você também pode ter um resultado muito pior se tomar decisões de implementação não tão ideais durante este processo. Além disso, recentemente as coleções foram adaptadas para execução paralela em _cores_ múltiplos. Coleções paralelas suportam as mesmas operações que as seqüenciais, de forma que não é necessário aprender novas operações e o código não precisa ser reescrito. Você pode converter uma coleção seqüencial em uma paralela simplesmente invocando o método `par`.
+
+**Universal:** Coleções provêm as mesmas operações sobre qualquer tipo onde faz sentido que elas existam, de forma que você possa construir muitas coisas com um vocabulário de operações relativamente pequeno. Por exemplo, uma _string_ é conceitualmente uma seqüência de caracteres. Consequentemente, em coleções Scala, _strings_ suportam todas as operações de seqüências. O mesmo vale para _arrays_.
```

Comment:
wrong conjugation of the "Prover" veb: `Coleções provêm`. correct: `Coleções proveem`.